docs/getting-started.md

```diff
@@ -232,7 +224,682 @@ curl localhost:10101/index/repository/query \
 Please note that while user ID 99999 may not be sequential with the other column IDs, it is still a relatively low number. 
 Don't try to use arbitrary 64-bit integers as column or row IDs in Pilosa - this will lead to problems such as poor performance and out of memory errors.
 
+#### Using Go
+
+Pilosa requires Go 1.12 or higher. It is also recommended that you have a code editor downloaded.
```

Comment:
This might be personal preference, but I think we can leave out the "code editor" recommendation.